eZ/Publish/Core/Search/Slot.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
- * File containing the Solr\Slot abstract class
+ * This file is part of the eZ Publish Kernel package
  *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  * @version //autogentag//
  */
 
-namespace eZ\Publish\Core\Search\Solr;
+namespace eZ\Publish\Core\Search;
```

Comment:
May I suggest we create a `Common` namespace or something?
If we do this we redo the mistake done in Persistence having several folders which are not implementation of a engine at root, messes up the convention of <API/SPI-part>/Implementation organization in Core.